.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
What is this launch program used for? I dont think this is needed when yarn or node launches it anyway from the package.json file?

Reply:
i dont know why it included this. i just checked a debug setting on my vscode and then this showed up. I need to delete it.